src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx

```diff
@@ -1,23 +1,190 @@
+  description?: string;
+};
 
 type Props = {
   organization: Organization;
 } & RouteComponentProps<{orgId: string}, {}>;
 
-class Relays extends AsyncComponent<Props> {
+type State = AsyncComponent['state'] & {
+  relays: Array<Relay>;
+};
+
+const relaysMock: Array<Relay> = [
+  {
+    public_key: '1:bb:6e:af:66:b4:38:e0:62:83:62:15:22:7',
+    name: 'First key',
+    description: 'optional description for the key',
+    first_seen: '2020-02-07T15:17:00Z',
+    last_seen: '2020-02-07T15:17:00Z',
+    created: '2020-02-07T15:17:00Z',
+  },
+  {
+    public_key: '2:bb:6e:af:66:b4:38:e0:62:83:62:15:22:7',
+    name: 'Second key',
+    description: 'optional description for the key',
+    first_seen: '2020-02-07T15:17:00Z',
+    last_seen: '2020-02-07T15:17:00Z',
+    last_modified: '2020-02-07T15:17:00Z',
+    created: '2020-02-07T15:17:00Z',
+  },
+];
+
+class Relays extends AsyncComponent<Props, State> {
+  getDefaultState() {
+    return {
+      ...super.getDefaultState(),
+      relays: relaysMock,
+    };
+  }
+
+  // TODO(Priscila): activate the code below as soon as the endpoint is provided
+  // getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+  //   return [['relays', `/organizations/${this.props.organization.slug}/relay-keys`]];
+  // }
+
+  handleDelete = (id: Relay['public_key']) => () => {};
+
+  handleAdd = () => {};
+
   renderBody() {
+    const {relays} = this.state;
     return (
       <React.Fragment>
         <SettingsPageHeader title={t('Relays')} />
+        <Panel>
+          <PanelHeader>{t('Relays')}</PanelHeader>
+          <PanelAlert type="info">
+            {tct('For more details, see [linkToDocs].', {
+              linkToDocs: (
+                <ExternalLink href={RELAY_DOCS_LINK}>
+                  {t('full Relay documentation')}
+                </ExternalLink>
+              ),
+            })}
+          </PanelAlert>
+          <PanelBody>
+            {relays.map(
+              ({public_key, name, created, last_seen, first_seen, last_modified}) => (
+                <Content key={public_key}>
+                  <Info>
+                    <InfoItem>
+                      <Name>{name}</Name>
+                    </InfoItem>
+                    <InfoItem>
+                      <PublicKey>{public_key}</PublicKey>
+                    </InfoItem>
+                    <InfoItem>
+                      <Time label={t('Created:')} date={created} />
+                    </InfoItem>
+                    <InfoItem>
+                      <Time label={t('First Seen:')} date={first_seen} />
+                    </InfoItem>
+                    <InfoItem>
+                      <Time label={t('Last Seen:')} date={last_seen} />
+                    </InfoItem>
+                    <InfoItem>
+                      <Time label={t('Last modified:')} date={last_modified} />
+                    </InfoItem>
+                  </Info>
+                  <Tooltip title={t('Delete Rule')}>
```

Comment:
You can use the `title` attribute on a button to add a tooltip with fewer lines of code.